controllers/openstackdataplanenode_controller.go

```diff
@@ -471,3 +478,13 @@ func (r *OpenStackDataPlaneNodeReconciler) GetServices(instance *dataplanev1beta
 	}
 	return instanceRole.Spec.NodeTemplate.Services
 }
+
+// checkDeployProtectionAnnotation checks to see if the node has the deploy protection annotation set. If this is set,
+// we will block the execution of Ansible tasks until it is cleared.
+func (r *OpenStackDataPlaneNodeReconciler) checkDeployProtectionAnnotation(instance *dataplanev1beta1.OpenStackDataPlaneNode, annotation map[string]string) bool {
+	for k := range instance.ObjectMeta.GetAnnotations() {
+		_, keyExists := annotation[k]
+		return keyExists
```

Comment:
ditto, an extended values check

Reply:
I did this intentionally because I thought it might be better to have users completely remove the annotation, rather than allowing true and false values. I just thought it might be less complex from a user experience perspective. But happy to get a few more inputs on this one and we can implement the agreed upon handling.